pages/static/dist/css/admin.css

```diff
@@ -0,0 +1,8 @@
+#result_list th, #result_list td {
```

Comment:
Why are we storing the compiled version in git? Shouldn't this be built on the fly and not stored in version control? Don't we get merge conflicts by storing the compiled versions in git?

Reply:
It's a holdover from our old infrastructure setup that we eventually need to fix. But yes, you're correct, we should not be storing compiled assets in git.